SampleObjC/sdk-ios-objc/OCProfileViewController.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ProfileViewController.h
+//  sdk-ios-objc
+//
+//  Created by Simon Hamadene on 16/10/2018.
+//  Copyright © 2018 Yoti Limited. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface OCProfileViewController : UIViewController
```

Comment:
I don't think `OC` prefix is proper and following the Apple's design guidelines

Reply:
@casperkclee : This is Objective C. I guess the naming convention came with Swift, am I wrong?

Reply:
Not really, I am talking about the naming convention for Objective C written by Apple
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Conventions/Conventions.html
> Two-letter prefixes like these are reserved by Apple for use in framework classes

While third-party developers (that’s us) opt for 3 letters or more.

Reply:
I understand. Thanks!